app/src/androidTest/java/com/google/samples/apps/sunflower/viewmodels/PlantDetailViewModelTest.kt

```diff
@@ -54,4 +66,18 @@ class PlantDetailViewModelTest {
     fun testDefaultValues() {
         assertFalse(getValue(viewModel.isPlanted))
     }
+
+    @Test
+    fun testSnackbarMessage_AddedPlantToGardenString() {
+        assertNotEquals(viewModel.snackbarText.value, R.string.added_plant_to_garden)
```

Comment:
It's better to use assertThat over other Assert Methods.

Main Reasons:
- Readability
- Better Failure Messages
-Type Safety

Ref: https://objectpartners.com/2013/09/18/the-benefits-of-using-assertthat-over-other-assert-methods-in-unit-tests/

I'm ready to put a new PR to change in the current code base.